module/zstd/zstd-in.c

```diff
@@ -41,6 +41,16 @@
 #define	ZSTD_LIB_DICTBUILDER 0
 #define	ZSTD_LIB_DEPRECATED 0
 #define	ZSTD_NOBENCH
+#define	ZSTD_NODICT
+#define	ZSTD_NO_INTRINSICS
+#define	ZSTD_NO_UNUSED_FUNCTIONS
```

Comment:
with this being added you can try to remove the stack-warning workarounds (Wframe-larger-than=20480) in `module/zstd/Makefile.in` and `lib/libzstd/Makefile.am`

Reply:
They should already be, let me check again.

Reply:
Module one is indeed removed in this PR already and the lib makefile has already the stack warning removed in a earlier PR by someone else... 👍